apps/rush-lib/src/logic/npm/NpmShrinkwrapFile.ts

```diff
@@ -120,13 +121,13 @@ export class NpmShrinkwrapFile extends BaseShrinkwrapFile {
   }
 
   /** @override */
-  public getWorkspaceKeys(): ReadonlyArray<string> {
-    throw new InternalError('Not implemented');
+  public getProjectShrinkwrap(project: RushConfigurationProject): BaseProjectShrinkwrapFile | undefined {
+    return undefined;
```

Comment:
If we don't have a per-project shrinkwrap, the incremental and cached builds should look at the repo-wide shrinkwrap. Can we confirm that we do this?